Sources/Application/Environment/AppEnvironment.swift

```diff
@@ -48,116 +77,146 @@ public final class AppEnvironment {
         return string
     }
     
-    static func updateReleaseFlags(_ flags: [String: Bool]) {
-        releaseFlags.flags = flags
+    class ReleaseFlags: FeatureFlagProvider {
+        var flags: [String: Bool] = [:]
+        
+        subscript(key: String) -> Any? {
+            flags[key]
+        }
     }
-}
-
-class ReleaseFlags: FeatureFlagProvider {
-    var flags: [String: Bool] = [:]
-
-    subscript(key: String) -> Any? {
-        flags[key]
+    
+    class FeatureFlags: FeatureFlagProvider {
```

Comment:
I think we should probably think about separating these out into another file.

Reply:
I guess the good thing about these being within the body of `AppEnvironment` is that they can't be referenced from outside it where they should never be needed, maybe within an extension in another file?